orcid-integration-test/src/test/java/org/orcid/integration/blackbox/api/v2/rc1/VerifyOrcidBeforeFetchElementTest.java

```diff
@@ -38,7 +38,7 @@
 import org.orcid.api.common.WebDriverHelper;
 import org.orcid.integration.api.pub.PublicV2ApiClientImpl;
 import org.orcid.jaxb.model.common_rc1.Visibility;
-import org.orcid.jaxb.model.error.OrcidError;
+import org.orcid.jaxb.model.error_rc2.OrcidError;
```

Comment:
@amontenegro i followed the same way we versioned other components.. use rc2 in all managers, everywhere.. except the ones which are specific to rc1.  is this class one of those?